layouts/partials/landingpage/product-cards.html

```diff
@@ -1,70 +1,83 @@
 <div class="row justify-content-center inner-row">
+	<!-- Studio Pro 9 card -->
 	<div class="card w-50">
 	  <div class="card-body">
 		<div class="row">
 		  <img class="my-icon" src="/icons/mx-pro-logo.png"><a href="/refguide/"><h4 class="card-title-w-50">Studio Pro 9</h4></a>
 		</div>
 		<p class="card-text"><ul class="landing-list">
-		  <li class="link-list"><a href="/refguide/general/">General Info</a></li>
+		  <!-- Change links HERE -->
+		  <li class="link-list"><a href="/refguide/general/">Studio Pro Overview</a></li>
 		  <li class="link-list"><a href="/refguide/modeling/">App Modeling</a></li>
 		  <li class="link-list"><a href="/refguide/mobile/">Mobile</a></li></ul></p>
 	  </div>
 	</div>
+	<!-- Studio 9 card -->
 	<div class="card w-50">
 	  <div class="card-body">
 		<div class="row">
 		  <img class="my-icon" src="/icons/mx-studio.png"><a href="/studio/"><h4 class="card-title-w-50">Studio 9</h4></a>
 		</div>
 		<p class="card-text"><ul class="landing-list">
+		  <!-- Change links HERE -->
 		  <li class="link-list"><a href="/studio/general/">General Info</a></li>
 		  <li class="link-list"><a href="/studio/page-editor/">Pages</a></li>
 		  <li class="link-list"><a href="/studio-how-to/">Studio How-to's</a></li></ul></p>
 	  </div>
 	</div>
   </div>
   <div class="row justify-content-center inner-row">
+	<!-- Developer Portal card -->
```

Comment:
This is also a bit ugly. I'd rather have the icon on the same row as "Developer", I think. (LOW PRIORITY)
![image](https://user-images.githubusercontent.com/35492184/157216060-3cf3ca53-d3ac-41da-a22e-2042a16ebc51.png)

Reply:
I will add this to the to do list and see. I agree it's not very nice.